### ISSUE TITLE
Historical Data import

### ISSUE THREAD

**Issue by MattSpeke** [yo8192/octo2influx#7]
Hi,

Have not installed yet. Does this pull historical data from Octopus? I would like to backdate my data from the last 12 months. Is it possible to export the data from the website and import to influxdb?

**Comment by MattSpeke** [comment]
I also have data from Solis Inverter and home assistant that I would like to import.

**Comment by yo8192** [comment]
> Does this pull historical data from Octopus?

Yes, it downloads the data from Octopus via their API for customers (rather than from their website). And yes they do retain I think at least one year of your data history.

> I also have data from Solis Inverter and home assistant that I would like to import.

That won't be covered by octo2influx, it is focused on downloading data from Octopus only.
